src/Arduino_WiFiConnectionHandler.cpp

```diff
@@ -67,7 +67,7 @@ NetworkConnectionState WiFiConnectionHandler::update_handleInit()
     return NetworkConnectionState::ERROR;
   }
 #if !defined(__AVR__)
```

Comment:
I think I've commented on this before, but you can really remove all those `#if !defined(__AVR__)` statements, since AVR is not going to be supported as a target platform, due to flash size constraints.

Reply:
yes, i think will do a general cleanup before releasing a new version of the library :wink: